### PR TITLE
Export metadata into S3 or Minio

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Export metadata into S3 or Minio

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: storage.go [hunk 1]
```diff
@@ -590,6 +590,54 @@ func (storage DBStorage) StoreTableMetadataIntoFile(fileName string, tableNames
...
+func (storage DBStorage) StoreTableMetadataIntoS3(ctx context.Context,
+	minioClient *minio.Client, bucketName string, objectName string,
+	tableNames []TableName) error {
+
+	buffer := new(bytes.Buffer)
+
+	// initialize CSV writer
+	writer := csv.NewWriter(buffer)
+
+	err := writer.Write([]string{"Table name", "Records"})
```
Comment (juandspy) [inline storage.go:604]: This code (until line 616) is duplicated from the previous function.

**Review by juandspy — CHANGES_REQUESTED** [review]
OK, but there is some duplication.

File: storage.go (hunk 1 above)
Comment (tisnik) [reply storage.go:604]: it is

Comment (tisnik) [reply storage.go:605]: updated

**Review by juandspy — APPROVED** [review]

**Pull request merged**